src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,121 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
+@SuppressWarnings("unchecked")
 public class ArrayList<T> implements List<T> {
+    private int maxListSize = 10;
+    private T[] values;
+    private int capacity;
+    private int size;
```

Comment:
useless variable
the `size` variable is enough
```suggestion
    private int size;
```

Reply:
I added this because Anton said there must be two variables for two different purposes. Or is it easier not to touch it on this stage?